client/blocks/eligibility-warnings/index.tsx

```diff
@@ -188,7 +188,8 @@ function getProceedButtonText( holds: string[], translate: LocalizeProps[ 'trans
 
 function isProceedButtonDisabled( isEligible: boolean, holds: string[] ) {
 	const resolvableHolds = [ 'NO_BUSINESS_PLAN', 'SITE_UNLAUNCHED', 'SITE_NOT_PUBLIC' ];
-	const canHandleHoldsAutomatically = union( resolvableHolds, holds ).length === 3;
+	const canHandleHoldsAutomatically =
+		[ ...new Set( [].concat( resolvableHolds, holds ) ) ].length === 3;
```

Comment:
Here, using `union` and `length` is quite an odd way to write this condition:
```js
holds.every( ( h ) => resolvableHolds.includes( h ) )
```

Reply:
Agreed, simplified in b79edbd8d8b9b3d1528ad3d4434f628d49729dfd.